2021-10-07 18:13:28,176 INFO    MainThread:96535 [wandb_setup.py:_flush():69] setting env: {}
2021-10-07 18:13:28,177 INFO    MainThread:96535 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-07 18:13:28,177 INFO    MainThread:96535 [wandb_init.py:_log_setup():357] Logging user logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211007_181328-23n4jrtc/logs/debug.log
2021-10-07 18:13:28,177 INFO    MainThread:96535 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211007_181328-23n4jrtc/logs/debug-internal.log
2021-10-07 18:13:28,178 INFO    MainThread:96535 [wandb_init.py:init():390] calling init triggers
2021-10-07 18:13:28,178 INFO    MainThread:96535 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-10-07 18:13:28,179 INFO    MainThread:96535 [wandb_init.py:init():435] starting backend
2021-10-07 18:13:28,179 INFO    MainThread:96535 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-07 18:13:28,198 INFO    MainThread:96535 [backend.py:ensure_launched():135] starting backend process...
2021-10-07 18:13:28,210 INFO    MainThread:96535 [backend.py:ensure_launched():140] started backend process with pid: 96562
2021-10-07 18:13:28,214 INFO    MainThread:96535 [wandb_init.py:init():440] backend started and connected
2021-10-07 18:13:28,222 INFO    MainThread:96535 [wandb_init.py:init():499] updated telemetry
2021-10-07 18:13:28,223 INFO    MainThread:96535 [wandb_init.py:init():522] communicating current version
2021-10-07 18:13:29,447 INFO    MainThread:96535 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-07 18:13:29,447 INFO    MainThread:96535 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-07 18:13:29,618 INFO    MainThread:96535 [wandb_init.py:init():562] starting run threads in backend
2021-10-07 18:13:31,851 WARNING MainThread:96535 [wandb_init.py:init():829] interrupted
Traceback (most recent call last):
  File "/opt/anaconda3/envs/rg/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 808, in init
    run = wi.init()
  File "/opt/anaconda3/envs/rg/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 565, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/opt/anaconda3/envs/rg/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 801, in communicate_run_start
    result = self._communicate(rec)
  File "/opt/anaconda3/envs/rg/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 539, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/anaconda3/envs/rg/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 77, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/anaconda3/envs/rg/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/anaconda3/envs/rg/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
